medusa/server/api/v2/show.py

```diff
@@ -86,6 +86,8 @@ def _handle_detailed_show(self, tv_show, query):
                 new, existing = tv_show.get_backlogged_episodes(allowed_qualities=allowed_qualities,
                                                                 preferred_qualities=preferred_qualities)
                 data = {'new': new, 'existing': existing}
+            elif query == 'archiveEpisodes':
```

Comment:
@OmgImAlexis i need to receive a yes|no if we archived episodes or not. so it can't be a POST